src/AuthService/AuthService.ts

```diff
@@ -0,0 +1,36 @@
+import firebase from "firebase/app";
```

Comment:
I don't see a change in `package.json` in this PR, is firebase already added in our dependecies?